### PR TITLE
[AAE-3345] - remove docker-registry module

### PR DESCRIPTION
**Pull request by SohelSaiyed**

### CODE REVIEW

File: examples/rancher-eks/variables.tf [hunk 1]
```diff
@@ -128,7 +123,7 @@ variable "identity_password_secret" {
 }
 
 variable "kubernetes_version" {
-  default     = 1.15
+  default     = 1.16
```
Comment (mteodori) [inline examples/rancher-eks/variables.tf:126]: should k8s change be in another ticket or was it just outdated?

File: examples/eks/variables.tf [hunk 2]
```diff
@@ -66,12 +61,12 @@ variable "enable_acs" {
 }
 
 variable "create_acs" {
-  default     = false
```
Comment (mteodori) [inline examples/eks/variables.tf:69]: these should stay false for paas as default, create_acs and create_identity, was this change intentional?

File: examples/rancher-eks/variables.tf (hunk 1 above)
Comment (SohelSaiyed) [reply examples/rancher-eks/variables.tf]: reverted back to 1.15

File: examples/eks/variables.tf (hunk 2 above)
Comment (SohelSaiyed) [reply examples/eks/variables.tf]: set it to false as default

**Review by mteodori — APPROVED** [review]
